<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>home</class>
 <widget class="QWidget" name="home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>10</y>
     <width>981</width>
     <height>631</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="terrain">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>80</y>
       <width>191</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Gestion terrain</string>
     </property>
    </widget>
    <widget class="QPushButton" name="employes">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>270</y>
       <width>191</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Gestion employees</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>250</y>
       <width>191</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="report">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>440</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Send report/email</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>-20</y>
       <width>311</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Gestion terrain</string>
     </property>
    </widget>
    <widget class="QPushButton" name="home_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>550</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Back to home</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom_stade">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_3">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>410</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>delete</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="adress_stade">
     <property name="geometry">
      <rect>
       <x>292</x>
       <y>120</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>184</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Capacité/chaises</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>200</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>nombres d'employes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="load_table_2">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>410</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>load data</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nb_emp">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>230</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="capacite">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_3">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>40</y>
       <width>491</width>
       <height>331</height>
      </rect>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom stade</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pdf_2">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>410</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Export to PDF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_2">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>410</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>add</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>adress</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Gestion emplyes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="home_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>490</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>back to home</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom_emp">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="prenom_emp">
     <property name="geometry">
      <rect>
       <x>302</x>
       <y>130</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="age_emp">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>240</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="salaire_emp">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>240</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>110</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Prenom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Age</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>210</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Salaire</string>
     </property>
    </widget>
    <widget class="QComboBox" name="post_emp">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>330</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Maintenance</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Securité</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Service medicale</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>300</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Poste</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>420</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_2">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>420</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>delete</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>50</y>
       <width>491</width>
       <height>331</height>
      </rect>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="load_table">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>420</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>load data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pdf">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>420</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Export to PDF</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QPushButton" name="home_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>500</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>back to home</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_sender">
     <property name="geometry">
      <rect>
       <x>342</x>
       <y>50</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_pass">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>120</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_smtp">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>160</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_465">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>200</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_recepiant">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>250</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_subject">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>310</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_text">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>370</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="sendBtn">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>460</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exitBtn">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>460</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>exit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>sender</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>130</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>pass</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>156</x>
       <y>170</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>smtp.gmail.com</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>200</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>465</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>250</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>to</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>310</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>sub</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>380</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>text</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
